cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -116,7 +118,19 @@ class Login_Management {
   }
 
   visitLoginPage () {
-    Cypress.config('baseUrl', Cypress.env('BRB'))
+    const instance = 'DEFAULT'
```

Comment:
wouldn't it make sense to keep this more flexible?
So you could give the state as a parameter to the method, for example loginManagement.visitLoginPage('DEFAULT')

Then it would be also usable for other states. 
II we change the tests to run on different states, we should keep it flexible, right?

Reply:
Right, But there are other changes which is necessary to do and I want to do it in separate ticket. Otherwise everything will be messed up.